paws.common/R/net.R

```diff
@@ -55,7 +55,7 @@ HttpResponse <- struct(
 # @param close Whether to immediately close the connection, or else reuse connections.
 # @param timeout How long to wait for an initial response.
 # @param dest Control where the response body is written
-new_http_request <- function(method, url, body = NULL, close = FALSE, timeout = NULL, dest = NULL) {
+new_http_request <- function(method, url, body = NULL, close = FALSE, timeout = NULL, dest = NULL, header = list()) {
```

Comment:
I believe we are missing the corresponding documentation for header

```
@param header blah blah
```

Do you mind updating the documentation for this? I believe you will need to run roxygen2 to create the doc for it.

Reply:
yes of course - this one slipped through